build/suffix-browser.js

```diff
@@ -6,3 +6,6 @@ this.sourceMap = {
   SourceMapGenerator: require('source-map/source-map-generator').SourceMapGenerator,
   SourceNode: require('source-map/source-node').SourceNode
 };
+if (module && module.exports) {
```

Comment:
This will break on the web, need to guard against the `module` binding not existing at all:

    if (typeof module === "object" && module && module.exports)